compiler/src/iree/compiler/Codegen/SPIRV/test/decompose_unit_matmul_transpose.mlir

```diff
@@ -0,0 +1,44 @@
+// RUN: iree-opt --split-input-file --iree-spirv-decompose-unit-matmul-transpose %s | FileCheck %s
```

Comment:
Please rename this file as well to match the name of the pass being tests. Also you probably need to change the RUN command.